### PR TITLE
[UwU] List changes

### PR DESCRIPTION
**Pull request by fennifith**
- Removes the bowtie styling from unordered lists
- Inherits the `<p>` font styling for list item content by default
  (this fixes empty `<li>text content</li>` having a different font size from `<li><p>text content</p></li>`)
- Removes the styling preventing nested lists from adding top/bottom margins
- Applies styling to `<ol>` ordered lists as well as unordered lists

### CODE REVIEW

File: src/styles/markdown/lists.scss [hunk 1]
```diff
@@ -14,32 +14,18 @@
 	);
```
Comment (crutchcorn) [inline src/styles/markdown/lists.scss:14]: Let's remove the `--list_bowtie` variables too? 😄

File: content/blog/test-edge-cases/index.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
+---
+{
+	title: "Test Content: Edge Cases",
+	description: "Look at all those edge cases!",
+	published: '2020-09-18',
```
Comment (crutchcorn) [inline content/blog/test-edge-cases/index.md:5]: Let's set this to `1999` or something so that we can easily detect if it makes it into prod

**Review by crutchcorn — APPROVED** [review]
